src/test/javascript/spec/helpers/sample/problemStatement.json

```diff
@@ -7,8 +7,8 @@
+    "problemStatementBubbleSortNotExecutedHtml": "<ol>\n<li><div class=\"pe-3-task-0 d-flex\" ng-version=\"{{ANGULAR_VERSION}}\"><div class=\"guided-tour\"><!--container--><!--container--><fa-icon size=\"lg\" class=\"ng-fa-icon test-icon text-secondary\" ng-reflect-size=\"lg\" ng-reflect-icon=\"[object Object]\"><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" data-prefix=\"fas\" data-icon=\"circle-question\" class=\"svg-inline--fa fa-circle-question fa-lg\" xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 512 512\"><path fill=\"currentColor\" d=\"M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM169.8 165.3c7.9-22.3 29.1-37.3 52.8-37.3l58.3 0c34.9 0 63.1 28.3 63.1 63.1c0 22.6-12.1 43.5-31.7 54.8L280 264.4c-.2 13-10.9 23.6-24 23.6c-13.3 0-24-10.7-24-24l0-13.5c0-8.6 4.6-16.5 12.1-20.8l44.3-25.4c4.7-2.7 7.6-7.7 7.6-13.1c0-8.4-6.8-15.1-15.1-15.1l-58.3 0c-3.4 0-6.4 2.1-7.5 5.3l-.4 1.2c-4.4 12.5-18.2 19-30.6 14.6s-19-18.2-14.6-30.6l.4-1.2zM224 352a32 32 0 1 1 64 0 32 32 0 1 1 -64 0z\"></path></svg></fa-icon><!--container--><span class=\"task-name\">Implement Bubble Sort</span><!--container--><span class=\"guided-tour test-status--linked text-secondary\">artemisApp.editor.testStatusLabels.totalTestsPassing: [{\"totalTests\":1,\"passedTests\":0}]</span><!--container--><!--container--></div></div> \nImplement the method <code>performSort(List&lt;Date&gt;)</code> in the class <code>BubbleSort</code>. Make sure to follow the Bubble Sort algorithm exactly. </li>\n<li><div class=\"pe-3-task-1 d-flex\" ng-version=\"{{ANGULAR_VERSION}}\"><div class=\"guided-tour success\"><fa-icon size=\"lg\" class=\"ng-fa-icon test-icon text-success\" ng-reflect-size=\"lg\" ng-reflect-icon=\"[object Object]\"><svg role=\"img\" aria-hidden=\"true\" focusable=\"false\" data-prefix=\"far\" data-icon=\"circle-check\" class=\"svg-inline--fa fa-circle-check fa-lg\" xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 512 512\"><path fill=\"currentColor\" d=\"M256 48a208 208 0 1 1 0 416 208 208 0 1 1 0-416zm0 464A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM369 209c9.4-9.4 9.4-24.6 0-33.9s-24.6-9.4-33.9 0l-111 111-47-47c-9.4-9.4-24.6-9.4-33.9 0s-9.4 24.6 0 33.9l64 64c9.4 9.4 24.6 9.4 33.9 0L369 209z\"></path></svg></fa-icon><!--container--><!--container--><!--container--><span class=\"task-name\">Implement Merge Sort</span><!--container--><span class=\"guided-tour test-status--linked text-success\">artemisApp.editor.testStatusLabels.totalTestsPassing: [{\"totalTests\":1,\"passedTests\":1}]</span><!--container--><!--container--></div></div> \nImplement the method <code>performSort(List&lt;Date&gt;)</code> in the class <code>MergeSort</code>. Make sure to follow the Merge Sort algorithm exactly.</li>\n</ol>",
```

Comment:
the only change here is changing "pe-task-0" to "pe-3-task-0" (programming exercise with id 3, and task with id 0)